controllers/spacebindingrequestmigration/spacebindingrequest_spacebinding_mapper.go

```diff
@@ -0,0 +1,53 @@
+package spacebindingrequestmigration
+
+import (
+	"context"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	commonconfig "github.com/codeready-toolchain/toolchain-common/pkg/configuration"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"k8s.io/apimachinery/pkg/types"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+// MapSpaceBindingRequestToSpaceBinding returns an event for the spacebinding that owns.
+func MapSpaceBindingRequestToSpaceBinding(cl runtimeclient.Client) func(spaceBindingREquest runtimeclient.Object) []reconcile.Request {
+	mapperLog := ctrl.Log.WithName("NSTemplateTierToSpaceMapper")
+	return func(obj runtimeclient.Object) []reconcile.Request {
+		if spaceBindingRequest, ok := obj.(*toolchainv1alpha1.SpaceBindingRequest); ok {
```

Comment:
it's just a detail, technically speaking, we don't need to cast the object and validate its type, right?

Reply:
correct! I've removed the cast in https://github.com/codeready-toolchain/host-operator/pull/922/commits/c86c05cade77cd893076a8a0a9088b0842015273